backend/src/routes/payments.js

```diff
@@ -46,18 +73,34 @@ const paymentReturn = async (req, res) => {
                 .status(422)
                 .json('Payment details not available in the system.');
         }
+
+        if (payment.payment_status) {
+            return res.status(422).json('This payment was already processed!.');
+        }
         await db.payments.updatePaymentStatus(ordernumber, req.query.SETTLED);
```

Comment:
We are getting two status from bambora for a successful payment.
SETTLED : {0,1}
0: Authorized
1: Completed

So I am updating whatever status we are getting for a successful payment so that this can be later used for analysis if something goes wrong.